graphjet-core/src/test/java/com/twitter/graphjet/algorithms/salsa/SalsaRequestBuilderTest.java

```diff
@@ -0,0 +1,43 @@
+package com.twitter.graphjet.algorithms.salsa;
+
+import com.google.common.base.Preconditions;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for class {@link SalsaRequestBuilder}.
+ *
+ * @date 2017-09-18
+ * @see SalsaRequestBuilder
+ **/
+public class SalsaRequestBuilderTest {
+    @Test
```

Comment:
The GraphJet codebase adopts a 2-space convention. Could you trim the spacing in this file?

Reply:
Done.